lib/palletjack/keytransformer.rb

```diff
@@ -251,6 +253,11 @@ def transform!(pallet)
             if self.respond_to?(transform.to_sym) then
               if new_value = self.send(transform.to_sym, param, context)
               then
+                new_value = TraceableString.new(new_value)
+                new_value.file = 'transformed'
+                new_value.line = 0
+                new_value.column = 0
+                new_value.byte = 0
```

Comment:
`Traceable` position for the transform rule that caused the value, so `transforms.yaml` would need to be loaded as a traceable structure.